application-task-ui/src/main/resources/TaskManager/GanttMacro/TaskUpdateService.xml

```diff
@@ -169,9 +163,6 @@
     <property>
       <allow>1</allow>
     </property>
-    <property>
-      <groups>XWiki.XWikiAllGroup</groups>
```

Comment:
You still have the `XWiki.XWikiRights` objects, even if there are no groups specified. Are these needed?